workflows/preprocess/preprocess.wdl

```diff
@@ -381,7 +381,11 @@ task counts_to_adata {
 	input {
 		String sample_id
 		String batch
+
+		# JAH hack here
 		String project_id
+		String dataset_id
```

Comment:
@ergonyc is this the ASAP_dataset_id in SAMPLE.csv?

Reply:
yes. or it could be the STUDY.team_dataset_id.  Either way its an ugly string, but we need to avoid collisions from the same team.